packages/backEnd/src/services/adds.services.ts

```diff
@@ -63,19 +58,14 @@ export class AddService {
   }
 
   async AddAdvertising(req: MulterRequest, res: Response, next: NextFunction) {
+    const whitelist = ["image/png", "image/jpeg", "image/jpg"];
     try {
-      if (req.files > 0) {
-        if (
-          req.files.mimetype !== "image/png" ||
-          req.files.mimetype !== "image/jpg" ||
-          req.files.mimetype !== "image/jpeg"
-        )
-          return res
-            .status(StatusCode.BAD_REQUEST)
-            .json({ error: "Only .png, .jpg and .jpeg format allowed!" });
-        // const spacesFiles = await uploadedFilesToSpaces(req.files);
-        return res.status(StatusCode.SUCCESS).json({ msg: 200 });
-      }
+      // console.log(req.files[0].mimetype);
```

Comment:
We shouldn't store unused code. If it is TODO, add this annotation.

Reply:
I will remove